### DISCUSSION TITLE
Don't let users disable Retrieval Step and Deletable Pushes

### DISCUSSION THREAD

**Discussion by Julian0o** [pglombardo/PasswordPusher#2822] [General]
Hi there,

i have configured the two options:
enable_retrieval_step: true
enable_deletable_pushes: true

But i don't want the users to disable these two options. Is it possible? I cannot find anything in the config for this...

Another Question: Is there a way to stop users from modifying the Password Generator settings? 

Thanks!

**Comment by pglombardo** [comment]
Hi @Julian0o - thanks for the request!

These options exist in the Premium/Pro version on pwpush.com but aren't available in the OSS edition (yet).

They will be added eventually but there is no ETA of yet.  I'll keep this request open and will update when the feature is added.

**Reply by Julian0o** [reply]
Thanks for the reply! Like to have it in the OSS edition in the Future!

**Reply by Julian0o** [reply]
If there will be a self-hosted Version of the Pro Version, I will look into it to make a subscription for our company!
